routes/index.js

```diff
@@ -1,9 +1,29 @@
 const express = require('express')
 const router = express.Router()
 const passport = require('passport')
+const db = require('../database/db.js')
 
 router.get('/', function(request, response){
-  request.url.length > 40 ? response.render('admin') : response.render('index')
+  if( request.url.length > 40 ){
+    db.displayInventoryItems()
+    .then( results => {
```

Comment:
Indentation is off.  Since these are single line functions, consider collapsing to one line to make this a little more concise.